tests/app/views/contexts/test_calculated_summary_context.py

```diff
@@ -9,237 +7,115 @@
-            context_summary["calculated_question"]["answers"][0]["value"], "9 cm"
-        )
-
-    def test_build_view_context_for_percentage_calculated_summary(self):
-        current_location = Location(
-            section_id="default-section", block_id="percentage-total-playback"
-        )
-
-        calculated_summary_context = CalculatedSummaryContext(
+            "9 cm",
+            2,
+            [],
+        ),
+        (
+            "percentage-total-playback",
+            "en_GB",
             "en",
-            self.schema,
-            self.answer_store,
-            self.list_store,
-            self.progress_store,
-            self.metadata,
-            self.response_metadata,
-        )
-
-        context = calculated_summary_context.build_view_context_for_calculated_summary(
-            current_location
-        )
-
-        self.assertIn("summary", context)
-        self.assert_summary_context(context)
-        self.assertEqual(len(context["summary"]), 6)
-        context_summary = context["summary"]
-        self.assertTrue("title" in context_summary)
-        self.assertEqual(
-            context_summary["title"],
             "We calculate the total of percentage values entered to be 20%. Is this correct?",
-        )
-
-        self.assertTrue("calculated_question" in context_summary)
-        self.assertEqual(
-            context_summary["calculated_question"]["title"],
-            "Grand total of previous values",
-        )
-        self.assertEqual(
-            context_summary["calculated_question"]["answers"][0]["value"], "20%"
-        )
-
-    @patch("app.jinja_filters.flask_babel.get_locale", Mock(return_value="cy"))
-    def test_build_view_context_for_number_calculated_summary(self):
-        current_location = Location(
-            section_id="default-section", block_id="number-total-playback"
-        )
-
-        calculated_summary_context = CalculatedSummaryContext(
+            "20%",
+            2,
+            [],
+        ),
+        (
+            "number-total-playback",
+            "cy",
             "cy",
-            self.schema,
-            self.answer_store,
-            self.list_store,
-            self.progress_store,
-            self.metadata,
-            self.response_metadata,
-        )
-
-        context = calculated_summary_context.build_view_context_for_calculated_summary(
-            current_location
-        )
-
-        self.assertIn("summary", context)
-        self.assert_summary_context(context)
-        self.assertEqual(len(context["summary"]), 6)
-        context_summary = context["summary"]
-        self.assertTrue("title" in context_summary)
-        self.assertEqual(
-            context_summary["title"],
             "We calculate the total of number values entered to be 22. Is this correct?",
-        )
+            "22",
+            2,
+            [],
+        ),
+    ),
+)
+def test_build_view_context_for_currency_calculated_summary_no_skip(
```

Comment:
Is the name right?. I don't see ones for `with_skip`?. Also, the answer to the skip block is `Yes`.